lib/deployServiceClient.js

```diff
@@ -89,7 +95,7 @@ const presignUrlsAndUploadFiles = () => {
         uploadFile(data.fileName, data.url)
           .then(() => {
             logger.Print('.');
-            uploaded[data.fileName] = data.accessUrl;
+            uploaded[data.fileName] = { url: data.accessUrl, mtime: ((new Date()).getTime()/1000|0) };
```

Comment:
I don't get the `|0` ? What does that do?

Reply:
round

Reply:
JavaScript is weird.